startup.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+chown root /edgeone/filebeat/filebeat.yml
```

Comment:
I added this like because it has some troubles while trying to use the script

Reply:
This issue is related with root switch